src/inspect_ai/_cli/trace.py

```diff
@@ -109,11 +109,13 @@ def anomolies_command(trace_file: str | None, filter: str | None, all: bool) ->
     canceled_actions: dict[str, ActionTraceRecord] = {}
     error_actions: dict[str, ActionTraceRecord] = {}
     timeout_actions: dict[str, ActionTraceRecord] = {}
+    start_trace: ActionTraceRecord | None = None
```

Comment:
@jjallaire, this one was a pretty good catch in my opinion. I _think_ I've satisfied `pylint` keeping the behavior as close as possible to what was previously intended. However, there definitely was at least one bug with respect to setting a locally scoped `start_trace` and erroneously expecting it to impact the out outer scope. Please review this one carefully.